### DISCUSSION TITLE
AudioWagon on 2024 VOLVO XC60 ???

### DISCUSSION THREAD

**Discussion by KeewayHE** [MoleMan1024/audiowagon#147] [Q&A]
Hi !
Im an owner of 2024 VOLVO XC60,
How can I install AudioWagon on my car without Google network? Is there any place to download APK file？
Thanks！ 

![Keeway VOLVO XC60](https://github.com/MoleMan1024/audiowagon/assets/172566657/95414de1-c9c6-495a-b4df-69009cad49bf)

**Comment by MoleMan1024** [comment]
The car must support Android Automotive OS (AAOS). You can only get the app via the Google Play Store. If you don't have the Google Play Store inside the car, you can likely not install the app, you would need to check what kind of app store this Volvo car is using (Harman Ignite, Faurecia App store, Huawei app store etc.).

Otherwise, you can get the source code here in Github and compile an apk yourself. However AFAIK there is no way to install such an APK in Volvo cars by yourself.

Also the car will need to support USB host mode, so I can read data from the USB port. I know that for example the recent Volvo EX 30 does not support this based on what users told me. I don't know about this Volvo XC60.

**Reply by KeewayHE** [reply]
Thanks for your reply🤝
